### PR TITLE
Update 2021 RFP with additional questions

### PR DESCRIPTION
**Pull request by reylejano**
This PR updates the 2021 RFP with additional questions and replies from the subproject group

### CODE REVIEW

**Comment by rficcaglia** [comment]
not assigned to me to review, but it does match what was discussed on the slack and zoom discussions so +1

**Comment by jlk** [comment]
LGTM!

The list of design pages might be either considered scary in it's depth, or refreshingly well documented. I'll go with option b. :)

File: sig-security/security-audit-2021/RFP.md [hunk 1]
```diff
@@ -136,3 +136,36 @@ Separately, any code released with or as part of the report needs to be under th
...
+### How long does the Working Group envision the engagement lasting and what is the latest date you can receive the deliverables?
+
+The latest date to receive deliverables will be negotiated with the selected vendor.
+
+### Which attack vectors are of most concern to the Working Group.
+
+1. The attack vector most concerned about is unauthenticated access to a cluster resulting in compromise of the [components in-scope](#project_goals_and_scope)
+2. Crossing cluster boundaries for multi-cluster configuration
+3. Crossing namespaces
+4. Any attack vector that exists against the components in scope
```
Comment (tabbysable) [inline sig-security/security-audit-2021/RFP.md:167]:
```suggestion
4. Any other attack vector that exists against the components in scope
```

File: sig-security/security-audit-2021/RFP.md [hunk 2]
```diff
@@ -136,3 +136,36 @@ Separately, any code released with or as part of the report needs to be under th
...
+
+### How long does the Working Group envision the engagement lasting and what is the latest date you can receive the deliverables?
+
+The latest date to receive deliverables will be negotiated with the selected vendor.
+
+### Which attack vectors are of most concern to the Working Group.
+
+1. The attack vector most concerned about is unauthenticated access to a cluster resulting in compromise of the [components in-scope](#project_goals_and_scope)
+2. Crossing cluster boundaries for multi-cluster configuration
+3. Crossing namespaces
```
Comment (tabbysable) [inline sig-security/security-audit-2021/RFP.md:166]: I understand what this means, but I worry non-Kubernetes-experts might not?

I keep trying to write something and I don't like any of the things I come up with.

"Crossing namespaces, an attacker with access being able to affect resources their credentials do not allow" maybe?

File: sig-security/security-audit-2021/RFP.md (hunk 1 above)
Comment (reylejano) [reply sig-security/security-audit-2021/RFP.md]: @tabbysable updated the PR with the suggested change 
> 4. Any other attack vector that exists against the components in scope

File: sig-security/security-audit-2021/RFP.md (hunk 2 above)
Comment (reylejano) [reply sig-security/security-audit-2021/RFP.md]: Updated the PR with the suggested change:
> 3. Crossing namespace boundaries, an authenticated attacker being able to affect resources their credentials do not directly allow

**Comment by reylejano** [comment]
@tabbysable added recommendations, ptal

**Pull request merged**